index.html

```diff
@@ -45,6 +45,342 @@ <h2 class="supporting">
+                                    </li>
+                                </ul>
+                            </div>
+
+                        </div>
+                        <div class="action">
+                            <a href="#">
+                                <button>
+                                    <h3>See Project</h3>
+                                </button>
+                            </a>
+                        </div>
+                    </div>
+                    <div class="card">
+                        <div class="mask-group">
+                            <div class="right-block">
+                                <div class="title">
+                                    Profesional Art Printing Data
+                                </div>
+                                <div class="description">
+                                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                    has
+                                    been the industry's standard
+                                </div>
+                                <ul class="tags">
+                                    <li class="html">
+                                        <span>html</span>
+                                    </li>
+                                    <li class="bootstrap">
+                                        <span>bootstrap</span>
+                                    </li>
+                                    <li class="ruby">
+                                        <span>Ruby</span>
+                                    </li>
+                                </ul>
+                            </div>
+
+                        </div>
+                        <div class="action">
+                            <a href="#">
+                                <button>
+                                    <h3>See Project</h3>
+                                </button>
+                            </a>
+                        </div>
+                    </div>
+
+
+                </div>
+            </div>
+        </section>
+        <!-- end of works section -->
+
+        <!-- start of about me section -->
+
+        <section class="about-me-section">
+            <div class="about">
+                <span class="title">
+                    About me
+                </span>
+                <p class="description">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through
+                    some of my work and
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact
+                    me.
+                </p>
+                <a href="#">
+                    <button>
+                        <h3>
+                            Get My Resume
+                        </h3>
+                    </button>
```

Comment:
- Good job implementing almost all the changes requested by the previous CR :muscle: but you didn't fully implement the last one. You have applied the enabled and hover button interaction styles but you haven't implemented the pressed styles for all the buttons. So I would suggest you do this as well so your project can be  complete :wink: 

- Your buttons when pressed
![image](https://user-images.githubusercontent.com/102699958/213869458-46662c3c-1a8f-4d90-9ef5-461bd3821bab.png)
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/213869493-5bc3fe13-064d-421d-9d2d-9cd0d3faff47.png)